defaults/preferences/zotero.js

```diff
@@ -208,5 +208,8 @@ pref("extensions.zotero.annotations.noteTemplates.title", "<h1>{{title}}<br/>({{
 pref("extensions.zotero.annotations.noteTemplates.highlight", "<p>{{highlight}} {{citation}} {{comment}}</p>");
 pref("extensions.zotero.annotations.noteTemplates.note", "<p>{{citation}} {{comment}}</p>");
 
+// PDF Reader
+pref("extensions.zotero.pdfReader.defaultState", '{"scale":"page-width","scrollMode":0,"spreadMode":0}');
```

Comment:
Thanks for implementing default view settings! It's nice that you'll be able to change the default, but I think the initial default should be "auto". In many cases, "auto" (Ctrl/Cmd-0) is just the same as "page-width". However, in some cases "auto" is just the better default option, in my opinion. Two examples:

1. Landscape documents, see https://github.com/mozilla/pdf.js/pull/5229. If you view the PDF file mentioned there, e.g., on a 16:9 screen in Firefox, it will look much better in "auto" than in "page-width" mode:

> Landscape documents, like slide presentations, are much easier to read if "Automatic Zoom" fits the slide height so the full slide is visible. Here is an example landscape slide presentation:
> 
> https://wiki.mozilla.org/images/5/55/MobileOpportunity.pdf

2. On larger monitors, the "auto" setting limits the zoom level to something reasonable. I'd guess, especially on ultra wide monitors, the PDF could be zoomed to a ridiculously large size in "page-width" mode. Two related discussions on the forums are [here](https://forums.zotero.org/discussion/93542/feature-request-automatically-resize-option) and [here](https://forums.zotero.org/discussion/comment/394624/#Comment_394624).

In most cases, "auto" will give the best user experience, I think. It also seems that "auto" is the default in Firefox.

Reply:
I had a look at what "auto" does, see [here](https://github.com/mozilla/pdf.js/blob/8629a55215f88e406c304ccd18dd4ac54f44d9ac/web/pdf_viewer.js#L1135-L1189). If you think that the current "auto" scale is sometimes too small, you could consider changing `MAX_AUTO_SCALE`, perhaps even making it another hidden preference. The current value is 1.25, but people might prefer different maximum zoom levels in "auto" scale. (Note that this is different from a fixed numerical zoom level.)